src/core/trulens/core/app.py

```diff
@@ -421,6 +422,18 @@ def tru(self) -> core_connector.DBConnector:
         pydantic.PrivateAttr(default_factory=dict)
     )
 
+    tokens: list[object] = []
```

Comment:
thanks! didn't know about the difference between the two before. 

`object` is chosen because that's the signature of the `attach` function in the OTEL context API

<img width="591" alt="Screenshot 2024-12-24 at 10 03 17 AM" src="https://github.com/user-attachments/assets/de290c32-2b21-44ae-8e41-03d7389df30e" />

Reply:
<img width="924" alt="Screenshot 2024-12-24 at 10 03 59 AM" src="https://github.com/user-attachments/assets/d04e4170-9cec-435a-a675-21361f68512b" />